art-common/pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = [
+    "setuptools>=65.5.1",  # not directly required, pinned by Snyk to avoid a vulnerability
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["artcommonlib"]
+
+[project]
+dynamic = ["version"]
+name="rh-art-commonlib"
+authors = [
+    {name = "AOS ART Team", email = "aos-team-art@redhat.com"},
+]
+description = "CLI tool for managing and automating Red Hat software releases"
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ["openshift"]
+license.file = "LICENSE"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Environment :: Console",
+    "Operating System :: POSIX",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Bug Tracking",
+    "Topic :: Software Development :: Build Tools",
+    "Topic :: Utilities",
+    "Natural Language :: English",
+]
+dependencies = [
+    "future",
```

Comment:
Think we're in the future since we dropped support of python2.

Reply:
right...